container/features/src/main/resources/features.xml

```diff
@@ -826,10 +826,19 @@
     <feature name="opennms-trapd-listener" description="OpenNMS :: Trapd :: Listener" version="${project.version}">
       <feature>camel-blueprint</feature>
       <feature>opennms-trapd</feature>
-
+       <bundle>mvn:org.opennms.features.minion/core-impl/${project.version}</bundle>
```

Comment:
We should depend on the "minion-core-api" feature instead of the core-impl bundle and the scv feature.